CommentAgent.py

```diff
@@ -362,7 +362,11 @@ def review_pr(self) -> bool:
                 logging.error("Error parsing response, try again")
                 continue
             except github.GithubException:
-                return False
+                error = traceback.format_exc()
+                logging.error(f"Github Exception:\n{error}")
+                self.GPT.add_message(
+                    "user", f"Your code failed with exception {error} try again"
+                )
```

Comment:
Consider catching more specific exceptions than a general `Exception` in `get_body_event_and_comments` to avoid masking different error types. Specify the raised error types for better debugging and error handling.